package.json

```diff
@@ -83,9 +83,8 @@
     "@angular/platform-browser": "16.2.4",
     "@angular/platform-browser-dynamic": "16.2.4",
     "@angular/router": "16.2.4",
-    "@ngneat/subscribe": "^3.0.2",
-    "@tanstack/react-query": "^4.12.0",
-    "@tanstack/react-query-devtools": "^4.12.0",
+    "@tanstack/react-query": "^5.4.3",
```

Comment:
There is no need to use the react version anymore. You can do what he did here:

https://github.com/TanStack/query/pull/6195/files#diff-d3075a62994bf33ff94f30a28f5caac8e1128726d8df1dc70ae0df7b6c209918